project-base/phpstan.neon

```diff
@@ -2,7 +2,40 @@ parameters:
     ignoreErrors:
         # Add ignored errors here as regular expressions, e.g.:
         # - '#PHPUnit_Framework_MockObject_MockObject(.*) given#'
-        - '#Undefined variable: \$undefined#'
+        -
+            message: '#Undefined variable: \$undefined#'
+            path: *
+        # Ignore annotations in generated code
+        -
+            message: '#(PHPDoc tag @(param|return) has invalid value (.|\n)+ expected TOKEN_IDENTIFIER at offset \d+)#'
+            path: %currentWorkingDirectory%/tests/ShopBundle/Test/Codeception/_generated/AcceptanceTesterActions.php
+        -
+            message: '#(PHPDoc tag @throws with type .+ is not subtype of Throwable)#'
+            path: %currentWorkingDirectory%/tests/ShopBundle/Test/Codeception/_generated/AcceptanceTesterActions.php
+        # In tests, we often grab services using container->get or access persistent refrences using $this->getReference
+        -
+            message: '#(Property (Shopsys|Tests)\\.+::\$.+ \(.+\) does not accept object.)#'
+            path: %currentWorkingDirectory%/tests/ShopBundle/*
+        # In data fixtures, we often access persistent references using $this->getReference
```

Comment:
This look awful lot similar to the rule above (both even mention accessing persistent references). Could they be unified?

Reply:
True, fixed in c735d3a7d